### PR TITLE
chore: rephrase parsing instructions in assessments

### LINKED ISSUE #6130: Parsing instructions can be confusing
## Describe the bug

The parsing instructions say: "Run the first bookmarklet in the browser tab containing your target page"
Users might not know how to run the bookmarklet in their page. 

**To Reproduce**

Go to assessment > Parsing > Parsing

## Expected behavior

Instead, I propose we have: "Run the first bookmarklet in the browser tab containing your target page by selecting the bookmarklet in the browser window with your target page" 

This is a bit wordy so suggestions are appreciated.

## Context (please complete the following information)

- OS Name & Version:  Windows
- AI-Web Version & Environment:  2.35.0 MV3 Insider
- Browser Version:  Chrome
- Target Page:  any

## Are you willing to submit a PR?

Yes

## Did you search for similar existing issues?

Yes

## Additional context

Observed while talking with a user

### PR DESCRIPTION
**Pull request by Pranav-yadav**
#### Details

chore: rephrase parsing instructions in assessments

as suggested under https://github.com/microsoft/accessibility-insights-web/issues/6130#issuecomment-1276871238
> Suggestion: maybe "Go to the browser tab containing your target page and, in that browser window, select the first bookmarklet"?

##### Motivation

closes #6130 

##### Context

Instructions shown to user during **Parsing Assessments**

currently;
https://github.com/microsoft/accessibility-insights-web/blob/7ef8ee7aa066968ef0085ab2a72d5505a397a83c/src/assessments/parsing/test-steps/parsing.tsx#L52-L56

#### Pull request checklist

- [x] Addresses an existing issue: #6130 
- [-] Ran `yarn null:autoadd`
- [-] Ran `yarn fastpass`
- [-] Added/updated relevant unit test(s) (and ran `yarn test`)
- [-] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [-] (UI changes only) Added screenshots/GIFs to description above
- [-] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

**Comment by Pranav-yadav** [comment]
@ferBonnin done the required changes! do review and merge :)

PS: reminder: add `hacktoberfest-accepted` label before merging 👍

**Review by waabid — APPROVED** [review]
LGTM! 😄
